sympy/polys/numberfields.py

```diff
@@ -450,6 +451,31 @@ def _minpoly_cos(ex, x):
     raise NotAlgebraic("%s doesn't seem to be an algebraic element" % ex)
 
 
+def _minpoly_tan(ex):
+    """
+    Returns the minimal polynomial of ``tan(ex)``
+    see https://github.com/sympy/sympy/issues/21430
+    """
+    x = Symbol('x', real=True)
```

Comment:
`x` should be given as a call parameter. Creating a new symbol may be confusing if it is also in use elsewhere.

Reply:
Sure, it is given as a parameter rather than declared in the function.
Thanks,